### ISSUE TITLE
grunt usemin did not replace <!-- build:js({.tmp,app}) scripts/app.min.js --> block

### ISSUE THREAD

**Issue by angelapper** [angelapper/MVC6.Material.Design.Starter#1]
```
  <!-- build:js({.tmp,app}) scripts/app.min.js -->
  <script src="app.module.js"></script>
  <script src="components/users/usercontroller.js"></script>
  <script src="components/users/userservice.js"></script>
  <script src="populate_template_cache.js"></script>
  <!-- endbuild -->
```

should be replace by app.min.{hash}.js automatically, but it did not work

**Comment by angelapper** [comment]
here is something I can not understand. although, I removed ({.tmp,app}) for searching, at beginning it did not fix the block replacement problem. So I tried to debug it based on http://stackoverflow.com/questions/26769093/grunt-usemin-not-replacing-reference-block-with-revved-file-line suggestion, can not find any clue. 30 minutes passed, I got tired, and put app.min.js block before lib.min.js block and after compile, I found both of them automatically replaced by usemin, and then I put app.min.js back, well it works like a magic.... then I lost...
